Source/ISHRemote/Trisoft.ISHRemote/ISHRemote.PesterSetup.ps1

```diff
@@ -9,7 +9,7 @@ Write-Host "Generating and Executing Import-Module Statement..."
 $project = (Split-Path -Parent $MyInvocation.MyCommand.Path).ToLower()
 $project = $project.Substring(0, $project.LastIndexOf("ishremote"))
 $project = ($project + "ishremote")
-Import-Module (Join-Path $project "\bin\debug\ishremote.psm1") -DisableNameChecking
+Import-Module (Join-Path $project "\bin\debug\ISHRemote") -DisableNameChecking
```

Comment:
Why is this removed, I seem to remember that forcing `.psm1` was required to make sure that from a file path the module is loading over the PS1 file and not directly the DLL

Reply:
@ddemeyer This is how it works for me in Powershell ISE to see objects properly formatted. I am not directly specifying dll file, just "ISHRemote". Can you double check how it works for you?

Reply:
Let's continue with your code... hopefully we both remember this if it has side effects :)